tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/payload/content/PayloadContentCommits.java

```diff
@@ -0,0 +1,45 @@
+package slacknotifications.teamcity.payload.content;
+
+import jetbrains.buildServer.serverSide.SRunningBuild;
+import jetbrains.buildServer.users.SUser;
+import jetbrains.buildServer.vcs.SVcsModification;
+import slacknotifications.teamcity.Loggers;
+import slacknotifications.teamcity.SlackNotificator;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class PayloadContentCommits {
+    List<Commit> commits;
+
+    public PayloadContentCommits() {
+        commits = new ArrayList<Commit>();
+    }
+
+    void populateCommits(SRunningBuild sRunningBuild) {
```

Comment:
make this method public since thats the only real thing this class does. (which is good)